api/api/version_image_api_test.go

```diff
@@ -64,6 +65,103 @@ func TestVersionImageController_GetImage(t *testing.T) {
 					VersionID: 1,
 					ImageRef:  "ghcr.io/caraml-dev/project-model:1",
 					Exists:    true,
+					JobStatus: models.ImageBuildingJobStatusSucceeded,
+				},
+			},
+		},
+		{
+			name: "success, image exists but job is not exists anymore",
+			vars: map[string]string{
+				"model_id":   "1",
+				"version_id": "1",
+			},
+			modelService: func() *mocks.ModelsService {
+				svc := &mocks.ModelsService{}
+				svc.On("FindByID", mock.Anything, models.ID(1)).Return(&models.Model{
+					ID:        models.ID(1),
+					ProjectID: models.ID(1),
+					Type:      "pyfunc",
+				}, nil)
+				return svc
+			},
+			versionService: func() *mocks.VersionsService {
+				svc := &mocks.VersionsService{}
+				svc.On("FindByID", mock.Anything, models.ID(1), models.ID(1), mock.Anything).Return(&models.Version{
+					ID:      models.ID(1),
+					ModelID: models.ID(1),
+				}, nil)
+				return svc
+			},
+			versionImageService: func() *mocks.VersionImageService {
+				svc := &mocks.VersionImageService{}
+				svc.On("GetImage", mock.Anything, mock.Anything, mock.Anything).
+					Return(models.VersionImage{
+						ProjectID: 1,
+						ModelID:   1,
+						VersionID: 1,
+						ImageRef:  "ghcr.io/caraml-dev/project-model:1",
+						Exists:    true,
+						JobStatus: models.ImageBuildingJobStatusUnknown,
```

Comment:
unknown is returned when job is missing?

Reply:
Yes, correct.